jerry-core/jrt/jrt.h

```diff
@@ -40,6 +40,18 @@
 # define __attr_pure___ __attribute__((pure))
 #endif /* !__attr_pure___ */
 
+#ifdef CONFIG_STORE_CONST_IN_ROM
+#ifndef ICACHE_RODATA_ATTR
+#define ICACHE_RODATA_ATTR  __attribute__((section(".irom.text")))
```

Comment:
Actually the whole mechanism should go to the build system:

    CFLAGS+=-DJERRY_CONST_DATA="__attribute__((aligned(4))) __attribute__((section(".irom.text")))"

And jrt should only check whether JERRY_CONST_DATA is defined. If not, it defines the macor as an empty string. The change in the config.h could also be removed in this case.